versioned_docs/version-v0.19/Overview/SupportedDataSources.md

```diff
@@ -9,83 +9,84 @@ sidebar_position: 5
+| GitHub              | Source Code Management, Code Review, Issue Tracking, CI/CD (GitHub Actions)  | Cloud                              | Available               | `github`, `gitextractor`      | Incremental Sync                                                    |
+| GitLab              | Source Code Management, Code Review, Issue Tracking, CI/CD (GitLab CI)       | Cloud, Community Edition 11+       | Available               | `gitlab`, `gitextractor`      | Full Refresh, Incremental Sync(for `issues`,`MRs`)                  |
+| Jira                | Issue Tracking                                                               | Cloud, Server/Data Center 7.x, 8.x | Available               | `jira`                        | Full Refresh, Incremental Sync(for `issues` and related)            |
+| Jenkins             | CI/CD                                                                        | 2.263.x+                           | Available               | `jenkins`                     | Incremental Sync                                                    |
+| BitBucket (Beta)    | Source Code Management, Code Review                                          | Cloud                              | Advanced Mode Available | `bitbucket`, `gitextractor`   | Full Refresh                                                        |
+| TAPD (Beta)         | Issue Tracking                                                               | Cloud                              | Advanced Mode Available | `tapd`                        | Full Refresh, Incremental Sync(for `stories`, `bugs`, `tasks`)      |
+| Teambition (Beta)   | Issue Tracking                                                               | Cloud                              | Advanced Mode Available | `teambition`                  | Full Refresh                                                        |
+| Zentao (Beta)       | Issue Tracking                                                               | v17.x, v18.x                       | Advanced Mode Available | `zentao`                      | Full Refresh                                                        |
+| Gitee (WIP)         | Source Code Management, Code Review, Issue Tracking                          | Cloud                              | Not Available           | `gitee`, `gitextractor`       | Full Refresh, Incremental Sync(for `issues`,`MRs`)                  |
+| PagerDuty           | Issue Tracking                                                               | Cloud                              | Available               | `pagerduty`                   | Incremental Sync                                                    |
+| Opsgenie           | Issue Tracking                                                               | Cloud                              | Available               | `opsgenie`                     | Full Refresh (for `users`,`teams`), Incremental Sync (for `issues`) |
```

Comment:
This may need to be deleted because this is version v0.19

Reply:
Sorry! I'll updated it